src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -482,3 +489,8 @@ class MaybeGraph:
     @property
     def failed(self):
         return len(self.problems) > 0
+
+
+# non-top-level import is required to avoid cyclic dependency
+# pylint: disable=wrong-import-position, cyclic-import
```

Comment:
Why can't magic command be part of this file? Cyclic import disable is not allowed, as it leads to a bigger mess 🚫